packages/styling/src/styles/UtilityStyles.ts

```diff
@@ -0,0 +1,12 @@
+import { IRawStyle } from '../interfaces/index';
```

Comment:
Are there plans to use UtilityStyles.ts for other future styles? If not, it may be more appropriate to rename this file to something along the lines of ScreenReaderStyles.ts